packages/core/types/src/http/workflow-execution/admin/entities.ts

```diff
@@ -1,11 +1,92 @@
+export type TransactionStepStatus =
```

Comment:
I got most of these types from admin, but not sure where to check to validate if they are correct